### PR TITLE
feat(admin): set up admin package

### LINKED ISSUE #151: Set up admin package
# PLEASE ONLY USE THIS IF THE OTHER TEMPLATES DON'T APPLY

I set up admin package.

### PR DESCRIPTION
**Pull request by tomoyakuroda**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues (PRs without this will not be accepted)
- Fixes #151 

## Changes
- Copied and pasted files from client package
- Removed unnecessary dependencies and components

## PR Checklist
- [ ] Does your PR contain relevant tests that pass?
- [ ] Is your code linted? `yarn lint` or `yarn lint:fix`
- [ ] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: packages/admin/src/components/AppHeader/AppHeader.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.menu {
```
Comment (tristanMatthias) [inline packages/admin/src/components/AppHeader/AppHeader.module.css:1]: Where is this file used?

File: packages/admin/src/containers/Me.container.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { useLazyQuery } from '@apollo/react-hooks';
```
Comment (tristanMatthias) [inline packages/admin/src/containers/Me.container.ts:1]: Can we make this and the Auth containers part of the `ui` library? Dodn't want to duplicate this logic across projects

File: packages/admin/src/images/logo-mark.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 50 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (tristanMatthias) [inline packages/admin/src/images/logo-mark.svg:1]: Move this to UI

File: packages/admin/src/images/logo.svg [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<svg viewBox="0 0 147 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (tristanMatthias) [inline packages/admin/src/images/logo.svg:1]: Move this to UI package

File: packages/admin/src/lib/apollo.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console, no-underscore-dangle */
```
Comment (tristanMatthias) [inline packages/admin/src/lib/apollo.ts:1]: Same with this one

**Comment by tomoyakuroda** [comment]
Thank you for your reviewing, @tristanMatthias !
I moved these files you mentioned to ui package.
Should I also refactor client package where these filed are used with this pull request, or create another pull request for client package refactoring?

**Pull request merged**